server/controllers/request.js

```diff
@@ -0,0 +1,95 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+const {
+  updateRequest,
+  checkQueryForEmptyFields,
+  validateQueryForNewRequest,
+} = require("../utils/helperFunctions");
+
+// @route GET /request/load
+// @desc Get list of requests for logged-in user
+// @access Private
+exports.loadRequests = asyncHandler(async (req, res, next) => {
+  const uid = req.user.id;
+  let requests;
+
+  if (uid) {
+    requests = await Request.find({
+      userId: uid,
+    });
+  }
+
+  res.status(200).json(requests);
+});
+
+// @route POST /request/new
```

Comment:
Just `POST /requests`

Reply:
It's generally not common to explicitly define `/new` in REST APIs, reason is the HTTP method tells what is going to happen. 🙂

Reply:
Gotcha, that makes sense. I've never really known what the preffered convention is since I've only ever really written routes for myself. Thanks for clearing it up for me!